src/main/java/edu/kis/vh/nursery/list/Node.java

```diff
@@ -2,11 +2,10 @@
 
 public class Node {
```

Comment:
Lack of encapsulation in Node class (Task 11 in "accessory i hermetyzacja")

Reply:
Still not fixed: "Lack of encapsulation in Node class (Task 11 in "accessory i hermetyzacja")"